libfreerdp/core/peer.c

```diff
@@ -103,6 +103,7 @@ static HANDLE freerdp_peer_virtual_channel_open(freerdp_peer* client, const char
 		peerChannel->channelId = mcsChannel->ChannelId;
 		peerChannel->mcsChannel = mcsChannel;
 		mcsChannel->handle = (void*)peerChannel;
+		InitializeCriticalSection(&peerChannel->writeLock);
```

Comment:
In this commit (`[core,server] lock FreeRDP_WTSVirtualChannelWrite`), this code here is still there from the old version (even though it is removed in a later commit). This should be removed from this commit here though, as it would break bisecting otherwise (fails to compile).

Reply:
ok, merged the last 2 commits